scripts/js/queries.js

```diff
@@ -40,7 +42,56 @@ function getDnssecConfig() {
   });
 }
 
+// Fetch database info (earliest timestamp, sizes, ...) from the API and
+// initialize related globals.
+function getDatabaseInfo() {
+  $.getJSON(document.body.dataset.apiurl + "/info/database", data => {
+    console.log(data);
+    // earliest_timestamp is provided in seconds since epoch
+    // We have two sources: earliest_timestamp_disk (on-disk) and earliest_timestamp (in-memory)
+    // Use whichever is smallest and non-zero
+    let diskTimestamp = Number(data.earliest_timestamp_disk);
+    let memoryTimestamp = Number(data.earliest_timestamp);
+
+    // Filter out zero/invalid timestamps
+    let validTimestamps = [diskTimestamp, memoryTimestamp].filter(ts => ts > 0);
```

Comment:
Fixing tests - Removing `console.log()` and using `const` where needed:

```suggestion
  $.getJSON(document.body.dataset.apiurl + "/info/database", data => {
    // earliest_timestamp is provided in seconds since epoch
    // We have two sources: earliest_timestamp_disk (on-disk) and earliest_timestamp (in-memory)
    // Use whichever is smallest and non-zero
    const diskTimestamp = Number(data.earliest_timestamp_disk);
    const memoryTimestamp = Number(data.earliest_timestamp);

    // Filter out zero/invalid timestamps
    const validTimestamps = [diskTimestamp, memoryTimestamp].filter(ts => ts > 0);
```